packages/migrate/src/helpers/transform-utils.ts

```diff
@@ -12,16 +12,31 @@ export function getInterfaceByName(
   return matched as ts.InterfaceDeclaration;
 }
 
+export function hasNameInImportClause(
+  importClause: ts.ImportClause | undefined,
+  importName: string
+): boolean {
+  if (importClause?.name?.getFullText().trim() === importName) {
+    return true;
```

Comment:
Can you add test case for this, please. 
And should we cover `import * from "something"` as well?

Reply:
That will be the complete solution I mentioned early, the test cases and other things. Accessing the properties for such use cases are quirky compared to when the imports are in curly braces.